utils/mysql-queries/getChannels.js

```diff
@@ -1,9 +1,14 @@
-export const getChannels = async (mysql, page, onlyAvailable, size) => {
-    const filter = onlyAvailable ? 'WHERE availability IS TRUE' : '';
+export const getChannels = async (mysql, page, size, filters) => {
     const offset = (page - 1) * size;
     const [results] = await mysql.query(`
         SELECT name, availability FROM channels
-        ${filter}
+        ${filters?.length
+            ? `WHERE ${filters.join(' AND ')}`
```

Comment:
Do not use string interpolation in SQL.

Reply:
I wrapped this in a [mysql.escapeId()](https://github.com/mysqljs/mysql?tab=readme-ov-file#escaping-query-identifiers).